pkg/kubefed2/unjoin.go

```diff
@@ -110,6 +114,10 @@ func (j *unjoinFederation) Complete(args []string) error {
 		j.clusterContext = j.ClusterName
 	}
 
+	if j.hostClusterName == "" && strings.ContainsAny(j.HostClusterContext, ":/") {
+		glog.Fatal("host-cluster-name must be set if the name of the host cluster context contains one of \":\" or \"/\"")
```

Comment:
It is preferred to return an error instead of directly `glog.Fatal()`

Reply:
Valid comments, will incorporate.